steps/appium-server/0.0.1/step.yml

```diff
@@ -0,0 +1,47 @@
+title: Appium Server
```

Comment:
`Run Appium Server` would be a better title: https://devcenter.bitrise.io/contributors/create-your-own-step/#the-title